app/assets/javascripts/analytics/init.js.erb

```diff
@@ -16,7 +16,7 @@
     cookieDomain: cookieDomain,
     allowLinker: true,
     // This is served by Fastly in production, and returns an empty 200 response
-    govukTrackerGifUrl: '<%= asset_path "/static/a" %>',
+    govukTrackerUrl: '<%= asset_path "/static/a" %>',
```

Comment:
To save me opening yet another PR, any chance you could remove the trailing comma? It's causing a (minor) error in IE11 when in compatibility mode.